### PR TITLE
add GitHub Actions ci.yml

### PR DESCRIPTION
**Pull request by sullis**
docs: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2017 Contributors to the Eclipse Foundation
```
Comment (arthurdm) [inline .github/workflows/ci.yml]: Needs to be 2019 (or in a few days, 2020 :) )

**Comment by arthurdm** [comment]
Thanks for the contribution @sullis 

There's an error showing up about the ECA not being found for your username.  Can you please fill that out?

**Comment by genie-microprofile** [comment]
Can one of the admins verify this patch?

**Comment by phillip-kruger** [comment]
Closing. Included with #428

**Pull request closed without merging**